docs/source/cli/lightning_cli_advanced_3.rst

```diff
@@ -137,44 +114,6 @@ Here is an example:
         --trainer.callbacks+=LearningRateMonitor \
         --trainer.callbacks.logging_interval=epoch
 
-Lightning provides a mechanism for you to add your own callbacks and benefit from the command line simplification
-as described above:
-
-.. code-block:: python
-
-    from pytorch_lightning.utilities.cli import CALLBACK_REGISTRY
-
-
-    @CALLBACK_REGISTRY
-    class CustomCallback(Callback):
-        ...
-
-
-    cli = LightningCLI(...)
-
-.. code-block:: bash
-
-    $  python ... --trainer.callbacks+=CustomCallback ...
-
-.. note::
-
-    This shorthand notation is also supported inside a configuration file. The configuration file
-    generated by calling the previous command with ``--print_config`` will have the full ``class_path`` notation.
```

Comment:
Should this still be mentioned?

Reply:
I added again a note but a bit different to avoid duplication.